src/utilities/extendSchema.js

```diff
@@ -280,6 +379,18 @@ export function extendSchema(
     });
   }
 
+  function extendArgsMap(
+    args: GraphQLArgument[],
+  ): GraphQLArgument[] {
+    return args.map(arg => ({
```

Comment:
I think it's better to use `keyValMap` here so the result of this function can be assigned to `GraphQLFieldConfig::args` without additional manipulations.